google-compute-engine/src/main/java/org/jclouds/googlecomputeengine/features/InstanceApi.java

```diff
@@ -250,7 +251,8 @@ Operation setDiskAutoDelete(@PathParam("instance") String instanceName,
    @MapBinder(BindToJsonPayload.class)
    Operation setScheduling(@PathParam("instance") String instanceName,
                            @PayloadParam("onHostMaintenance") Scheduling.OnHostMaintenance onHostMaintenance,
-                           @PayloadParam("automaticRestart") boolean automaticRestart);
+                           @PayloadParam("automaticRestart") boolean automaticRestart,
+                           @PayloadParam("preemptible") boolean preemptible);
```

Comment:
@nacx **boop boop**...breaking change here. If we wanted to keep a backwards-compatible version of `setScheduling` too, how would we do that?